src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private static final Color[] COLOR_VALUES = Color.values();
+    private Random random = new Random();
```

Comment:
Avoid using non-final fields for utilities like Random that can be final, as they are not supposed to change once set.